src/main/resources/default/templates/biz/login.html.pasta

```diff
@@ -44,7 +44,11 @@
                                        required
                                        autocomplete="off"
                                        spellcheck="false"
-                                       class="form-control input-block-level"/>
+                                       class="form-control input-block-level pe-5"/>
+                                <span id="passwordToggleButton">
+                                    <i class="fa fa-eye position-absolute" id="passwordToggleIcon"
+                                       style="top: 50%; right: 10px; transform: translateY(-50%); cursor: pointer;"></i>
+                                </span>
```

Comment:
In my opinion using an input-addon would be the more "Bootstrap-way" but its okay.
Could `align-content: center` be used instead of the transform?

Reply:
<img width="676" alt="Bildschirmfoto 2024-12-04 um 11 08 55" src="https://github.com/user-attachments/assets/0880ef22-1d82-4060-88ac-ddc225954778">
...maybe,.... please react with 👍 or 👎 if i should switch